birdhouse/components/proxy/default.env

```diff
@@ -8,7 +8,7 @@ export PROXY_READ_TIMEOUT_VALUE="240s"
 
 # Content of "location /" in file config/proxy/conf.d/all-services.include.template
 # Useful to have a custom homepage.
-export PROXY_ROOT_LOCATION="return 302 https://\$host/jupyter/hub/login;"
+export PROXY_ROOT_LOCATION="return 302 https://\$host/magpie;"
```

Comment:
I saw that the same variable is set by default when enabling `jupyterhub`. 
Should we have a better strategy to define the desired root location redirect? 
It seems that which one will be applied depends on a combination of dependency resolution order + file-system listing order. I'm not sure if this is replicable, unless one explicitly overrides the variable in their `env.local`.

Reply:
> It seems that which one will be applied depends on a combination of dependency resolution order + file-system listing order

If jupyterhub is enabled it will always override the magpie settings in `default.env` since magpie is enabled in `DEFAULT_CONF_DIRS` which will always be read before `EXTRA_CONF_DIRS` (where jupyterhub is enabled). Because jupyterhub's `default.env` will be read second, it will override the variable set in magpie's `default.env`

Reply:
Ok. Makes sense. I would add a comment that explicit this assumption, and indicate that this is only as last resort default if nothing else was enabled. Realistically, Magpie should probably not be the entrypoint, but we need "something" to fill the variable.

Reply:
Add this comment about magpie as landing page is not the most user-friendly location to the var  `PROXY_ROOT_LOCATION` in `env.local.example` as well.